environments/template/group_vars/all.yml

```diff
@@ -13,3 +13,33 @@ timezone: Europe/Amsterdam
 # Default user to for Ansible "remote_user"
 # You can override this in groups_vars, or host_vars
 ansible_remote_user: root
+
+install_prefix: /usr/local
+
+db:
+  root_password: "{{ lookup('file',inventory_dir+'/private/password/db_root_password') }}"
+  root_user: root
+
+common:
+  ip: "*"
+  port: 443
+  ssl:
+    enabled: yes
+    certificate: "{{ lookup('file',inventory_dir+'/private/ssl_cert/webserver.crt') }}"
+    selfsignedcertificate: yes
+    key: "{{ lookup('file',inventory_dir+'/private/ssl_cert/webserver.key') }}"
+  web:
+    domain: lists.example.com
+    admin: support@renater.fr
+  mail:
+    domain: lists.example.com
+  admins:
+    - david@webmail.com
```

Comment:
those should be example emails

Reply:
Oh yes. Forgot them.
(actually, "webmail.com" does not seem to be a thing but still).

Reply:
This was fixed by commit ecc43e6fe1d2b98743bb952c8c7f9343159b2c62.